ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -18,12 +18,20 @@
 
 import com.netflix.archaius.api.annotations.Configuration;
 import com.netflix.archaius.api.annotations.DefaultValue;
+import com.netflix.archaius.api.annotations.PropertyName;
+import com.netflix.ndbench.api.plugin.common.NdBenchConstants;
 
 /**
  * @author vchella
  */
 @Configuration(prefix = "ndbench.config")
+
+// TODO - consider this.. eliminates a harded coded string in favor of rolling up to constant
+//@Configuration(prefix = NdBenchConstants.PROP_PREFIX)
 public interface IConfiguration {
+    public static String WRITE_RATE_LIMIT = "writeRateLimit";
+    public static String WRITE_RATE_LIMIT_FULL_NAME = NdBenchConstants.PROP_PREFIX + "writeRateLimit";
```

Comment:
Hi .. i added a comment for this constant (and moved as per vinay' request):

    // Use constant so as to avoid hard coded string references in calling code
 
here is the code that ref's this constant ->   
(in ndbenchdriver):
    public void updateWriteRateLimit(double newLimit) {
        settableConfig.setProperty(NdBenchConstants.WRITE_RATE_LIMIT_FULL_NAME, (int) Math.ceil(newLimit));
        onWriteRateLimitChange();
    }